src/lms-ds-loader/edfi_lms_ds_loader/Sql_lms_operations.py

```diff
@@ -139,26 +148,9 @@ def insert_new_records_to_production(self, table: str, columns: List[str]) -> No
 
         column_string = ",".join([f"\n    {c}" for c in columns])
 
-        statement = f"""
-INSERT INTO
-    lms.{table}
-({column_string}
-)
-SELECT{column_string}
-FROM
-    lms.stg_{table} as stg
-WHERE
-    NOT EXISTS (
-        SELECT
-            1
-        FROM
-            lms.{table}
-        WHERE
-            SourceSystemIdentifier = stg.SourceSystemIdentifier
-        AND
-            SourceSystem = stg.SourceSystem
-    )
-"""
+        statement = ""
+        if self.engine == DbEngine.MSSQL:
+            statement = MS_builder.insert_new_records_to_production(table, column_string)
```

Comment:
I think the SQL statement above will work on both systems, or could be rewritten to work on both.